tests/utils/minishift_latest_version.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+import requests
+import re
+import os
+
+OS = os.getenv('OS', 'linux')
```

Comment:
OS is not set on OS X I think, there is OSTYPE, but that seems to be _darwin15_ in my case.